vortex-duckdb/cpp/table_function.cpp

```diff
@@ -50,7 +50,7 @@ struct CTableGlobalData final : GlobalTableFunctionState {
     unique_ptr<vortex::CData> ffi_data;
 
     idx_t MaxThreads() const override {
-        return GlobalTableFunctionState::MAX_THREADS;
+        return 1;
```

Comment:
shall we hook this into the rust ext?

Reply:
probably better to fix forward if we can than landing this